doc/10-package-json-reference.md

```diff
@@ -0,0 +1,56 @@
+
+Hyperproof recommends following [semver](https://semver.org/) versioning when making changes to custom Hypersync apps.
+
+More information on the version attribute can be found [here](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#name).
+
+## app_hyperproof
+
+The `app_hyperproof` value in `pakage.json` is an object that contains values that Hyperproof uses to properly expose and run your custom Hypersync app. The object must contain the values in the table below.
+
+| Attribute               | Description                                                                                                                                                                                                                                                                              |
+| ----------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| name                    | The name of your application as shown in the Hyperproof UI.                                                                                                                                                                                                                              |
+| appType                 | The type of custom app. Must be set to `hypersync`.                                                                                                                                                                                                                                      |
+| authType                | The type of authentication/authorization used by your application. Acceptable values are `custom` or `oauth`.                                                                                                                                                                            |
+| category                | The category of your custom Hypersync app as shown in the Add Hypersync wizard. This value is also shown under Connected Accounts when viewing connection information. Can be any string value but it is recommended to reuse an existing category if applicable.                        |
+| descriptionCapabilities | Description shown to the user when inspecting a connection under Connected Accounts.                                                                                                                                                                                                     |
+| descriptionConnected    | Description shown to the user when inspecting a connection under Connected Accounts.                                                                                                                                                                                                     |
```

Comment:
Is there anything to differentiate these?

Reply:
No.  We have a bug tracking the fact that these are super redundant.  Hope to make one optional in this sprint.